eva-lib/src/main/java/uk/ac/ebi/eva/lib/models/ga4gh/GACall.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.lib.models.ga4gh;
+
+import java.util.Arrays;
+
+public class GACall {
```

Comment:
It wasn't documented originally, no idea, same for the other undocumented classes.